packages/eslint/test/test.mjs

```diff
@@ -2,34 +2,44 @@ import fs from 'fs';
 
 import { createRequire } from 'module';
 
+import { resolve } from 'path';
+
+import { fileURLToPath } from 'url';
+
 import test from 'ava';
 import nodeResolve from '@rollup/plugin-node-resolve';
 import { rollup } from 'rollup';
 
 import eslint from 'current-package';
 
+const fixtures = resolve(fileURLToPath(new URL('.', import.meta.url)), 'fixtures');
+
+test.before(() => {
+  process.chdir(fixtures);
+});
+
 test('should lint files', async (t) => {
   let count = 0;
   await rollup({
-    input: './test/fixtures/undeclared.js',
+    input: 'undeclared.js',
     plugins: [
       eslint({
         formatter: (results) => {
-          count += results[0].messages.length;
-          // eslint-disable-next-line prefer-destructuring
-          const { message } = results[0].messages[0];
-          t.is(message, "'x' is not defined.");
```

Comment:
we don't modify existing tests - that presents a surface area for regressions. instead, we create new tests with the new conditions we'd like to test. I doubt that ESLint would have introduced a breaking change in an 8.x minor version, so this means that the conditions of the test changed. [unless I'm mistaken in some way] please revert this one and create a new test for the new messages and conditions you'd like to test.

Reply:
sure, I'll change these later